src/mailer/index.ts

```diff
@@ -0,0 +1,45 @@
+import * as nodemailer from 'nodemailer';
+import {
+  signinMessage,
+  resetMessage,
+} from './templates';
+const config = {
+  service: process.env.MAIL_SERVICE,
+  auth: {
+    user: process.env.MAIL_USER,
+    pass: process.env.MAIL_PASS,
+  },
+};
+const transporter = nodemailer.createTransport(config);
+
+function sendEmail(body: any): Promise<any> {
+  return new Promise((res, rej) => {
+    transporter.sendMail(body, (err, info) => {
```

Comment:
nope, I passed callback to `sendMail`, it's not return any `Promise`, I resolved/rejected the `Promise` inside callback.